W-16451588 auto update duke (#348)
mulesoft/docs-code-builder: modules/ROOT/pages/_partials/acb-start-desktop.adoc

+
For versioning information about extensions in Anypoint Extension Pack, see the https://marketplace.visualstudio.com/items/salesforce.mule-dx-extension-pack/changelog[Change Log^].

> . Recommended: Ensure that the Anypoint extensions update automatically by right-clicking each extension, determining whether *Auto Update All (From Publisher)* is checked, and if not, selecting *Auto Update* for each extension, for example:

Reviewer: ```suggestion
. Recommended: Ensure that the Anypoint extensions update automatically by right-clicking each extension to see if *Auto Update All (From Publisher)* is checked. If it's not, select *Auto Update* for each extension, for example:
```

+
image::start-extensions-auto-update.png["Setting an extension to auto-update"]
+

> Make sure this option is checked. 

Reviewer: ```suggestion
Ensure this option is checked. 
```
